torch/csrc/jit/script/schema_matching.cpp

```diff
@@ -332,6 +332,8 @@ c10::optional<MatchedSchema> tryMatchSchema(
       // Argument has a default value and no value was provided, so use the
       // default
       actual_named_value = NamedValue(*arg.default_value());
+    } else if (arg.type()->cast<OptionalType>()) {
+      actual_named_value = NamedValue(IValue());
```

Comment:
Add test?

Reply:
Doesn't this mean that optionals always have default values? It's not something that we would want.

Reply:
Added a test, not entirely sure if it's the right place.